jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/OuterPanel.kt

```diff
@@ -0,0 +1,423 @@
+                    }
+                    row {
+                        browserLink(message("caws.getstarted.panel.link_text"), CawsEndpoints.CAWS_SPACES_DOC)
+                    }
+                }
+            }
+        )
+
+        border = IdeBorderFactory.createRoundedBorder().apply {
+            setColor(CodeWhispererColorUtil.POPUP_BUTTON_BORDER)
+            preferredSize = Dimension(Paneltext.PANEL_WIDTH, Paneltext.PANEL_HEIGHT)
+        }
+    }
+}
+
+class ResourceExplorerPanel : BorderLayoutPanel() {
+    init {
+        addToCenter(
+            panel {
+                indent {
+                    row {
+                        label(message("aws.getstarted.resource.panel_title")).bold()
+                            .applyToComponent {
+                                foreground = Paneltext.TITLE_TEXT_FONTCOLOR
+                            }
+                    }
+                    row {
+                        panel {
+                            row {
+                                label("Image").applyToComponent {
+                                    border = BorderFactory.createLineBorder(Paneltext.TEXT_FONTCOLOR)
+                                    preferredSize = Dimension(250, 110)
+                                }
+                            }
+                        }
+                    }
+
+                    row {
+                        text(message("aws.getstarted.resource.panel_description")).applyToComponent { foreground = Paneltext.TEXT_FONTCOLOR }
+                    }
+
+                    row {
+                        browserLink(
+                            message("codewhisperer.gettingstarted.panel.learn_more"),
+                            url = "https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/working-with-aws.html"
+                        )
+                    }
+
+                    row {
+                        button(message("aws.onboarding.getstarted.panel.button_iam_login")) {}.apply {
+                            applyToComponent {
+                                putClientProperty(DarculaButtonUI.DEFAULT_STYLE_KEY, true)
+                            }
+                        }
+                        topGap(TopGap.MEDIUM)
+                    }
+
+                    row {
+                        label(message("aws.getstarted.resource.panel_question_text")).applyToComponent { foreground = Paneltext.TEXT_FONTCOLOR }
+                    }
+                    row {
+                        browserLink(message("aws.onboarding.getstarted.panel.signup_iam_text"), url = "https://aws.amazon.com/free/")
+                    }
+                }
+            }
+        )
+
+        border = IdeBorderFactory.createRoundedBorder().apply {
+            setColor(CodeWhispererColorUtil.POPUP_BUTTON_BORDER)
+            preferredSize = Dimension(Paneltext.PANEL_WIDTH, Paneltext.PANEL_HEIGHT)
+        }
```

Comment:
can we declare a base class that declares this once so we're not copying this everywhere?